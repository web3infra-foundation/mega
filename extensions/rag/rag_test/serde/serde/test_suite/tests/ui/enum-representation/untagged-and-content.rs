use serde_derive::Serialize;

#[derive(Serialize)]
#[serde(untagged)]
#[serde(content = "c")]
enum E {
    A(u8),
    B(String),
}

fn main() {}
