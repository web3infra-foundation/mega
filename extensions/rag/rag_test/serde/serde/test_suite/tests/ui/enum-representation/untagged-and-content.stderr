error: untagged enum cannot have #[serde(content = "...")]
 --> tests/ui/enum-representation/untagged-and-content.rs:4:9
  |
4 | #[serde(untagged)]
  |         ^^^^^^^^

error: untagged enum cannot have #[serde(content = "...")]
 --> tests/ui/enum-representation/untagged-and-content.rs:5:9
  |
5 | #[serde(content = "c")]
  |         ^^^^^^^
